MobileCenterAnalytics/MobileCenterAnalytics/Internals/MSAnalyticsInternal.h

```diff
@@ -23,7 +25,7 @@
  * @param pageName  page name.
  * @param properties dictionary of properties.
  */
-+ (void)trackPage:(NSString *)pageName withProperties:(NSDictionary *)properties;
++ (void)trackPage:(NSString *)pageName withProperties:(nullable NSDictionary<NSString *, NSString *> *)properties;
```

Comment:
To fix the nullable warnings we had here I added NS_ASSUME_NONNULL_BEGIN/END but doing so I had to explicitly specify nullable parameters.